glide-core/redis-rs/Makefile

```diff
@@ -0,0 +1,96 @@
+
+	@echo "===================================================================="
+	@echo "Testing Connection Type TCP with all features and RESP2"
+	@echo "===================================================================="
+	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=tcp RUST_BACKTRACE=1 cargo test --locked -p redis --all-features -- --nocapture --test-threads=1 --skip test_module
+
+	@echo "===================================================================="
+	@echo "Testing Connection Type TCP with all features and RESP3"
+	@echo "===================================================================="
+	@REDISRS_SERVER_TYPE=tcp PROTOCOL=RESP3 cargo test -p redis --all-features -- --nocapture --test-threads=1 --skip test_module
+
+	@echo "===================================================================="
+	@echo "Testing Connection Type TCP with all features and Rustls support"
+	@echo "===================================================================="
+	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=tcp+tls RUST_BACKTRACE=1 cargo test --locked -p redis --all-features -- --nocapture --test-threads=1 --skip test_module
+
+	@echo "===================================================================="
+	@echo "Testing Connection Type TCP with all features and native-TLS support"
+	@echo "===================================================================="
+	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=tcp+tls RUST_BACKTRACE=1 cargo test --locked -p redis --features=json,tokio-native-tls-comp,connection-manager,cluster-async -- --nocapture --test-threads=1 --skip test_module
+
+	@echo "===================================================================="
+	@echo "Testing Connection Type UNIX"
+	@echo "===================================================================="
+	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=unix RUST_BACKTRACE=1 cargo test --locked -p redis --test parser --test test_basic --test test_types --all-features -- --test-threads=1 --skip test_module
+
+	@echo "===================================================================="
+	@echo "Testing Connection Type UNIX SOCKETS"
+	@echo "===================================================================="
+	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=unix RUST_BACKTRACE=1 cargo test --locked -p redis --all-features -- --test-threads=1 --skip test_cluster --skip test_async_cluster --skip test_module --skip test_cluster_scan
+
+	@echo "===================================================================="
+	@echo "Testing async-std with Rustls"
+	@echo "===================================================================="
+	@RUSTFLAGS="-D warnings" REDISRS_SERVER_TYPE=tcp RUST_BACKTRACE=1 cargo test --locked -p redis --features=async-std-rustls-comp,cluster-async -- --nocapture --test-threads=1 --skip test_module
```

Comment:
I don't think async-std is relevant

Reply:
Ofc it doesn't. But this PR is not about cleaning up redis-rs code. It only adds it